src/patternfly/components/Tabs/tabs-item.hbs

```diff
@@ -9,7 +9,10 @@
     {{/if}}
   {{/if}}
   {{~#if tabs-item--IsAdd}} pf-m-add{{/if}}
+  {{~#if tabs-item--IsOverflow}} pf-m-overflow{{/if}}
+  {{~#if tabs-item--IsOverflowExpanded}} pf-m-expanded{{/if}}
```

Comment:
I think these classes are OK on the item, though you could argue that they belong on the tab button (either move them or duplicate them). This is the safest approach, since it allows us to either style the item or the tab button in the overflow/expanded state, but it will add an extra layer of specificity to any of the button rules if we have to target it like `.pf-c-tabs__item.pf-m-[overflow/expanded] .pf-c-tabs__link {}` instead of just being able to do `.pf-c-tabs__link.pf-m-[overflow/expanded]`. Though we can keep what you have and maintain specificity at "20" if the changes to an overflow/expanded link happen with vars on `.pf-c-tabs__item.pf-m-[overflow/expanded]`. So I'd say keep it the way it us but curious your thoughts, also cc @mattnolting

Reply:
To be completely honest I don't have the expertise to have a strong definitive opinion.

I can say that the reason I defaulted to just applying these classes to the tabs item itself is that from a React perspective with applying these classes I think I would *expect* that the classes would be applied directly to the list item.

But since disabled is applied to the button element within the tab currently I think that having them on the button instead would also make sense.